frontend/src/pages/user/buyer/info/index.tsx

```diff
@@ -1,21 +1,135 @@
-import { useState } from 'react';
+import { useRef, useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { useForm, SubmitHandler } from 'react-hook-form';
 
-import TButton from '@components/TButton';
-import InfoItem from '@components/InfoItem';
+interface BuyerInfoProps {
+  name: string;
+  email: string;
+  address: string;
+  image: File | undefined;
+}
+
+const BuyerItemStyle = {
+  borderRadius: '10px',
+  padding: '10% 10% 5% 10%',
+  border: '1px solid var(--button_border, #34977F)',
+  background: 'var(--button_dark, #135142)',
+};
+
+const userImgStyle = {
+  borderRadius: '50%',
+  width: '100px',
+  height: '100px',
+  cursor: 'pointer',
+  boxShadow: '2px 4px 10px 2px rgba(0, 0, 0, 0.25)',
+};
+
+const labelStyle = {
+  fontSize: '16px',
+  fontWeight: '500',
+  color: '#FFFFFFAA',
+};
```

Comment:
color codes typically only have 6 digits perhaps there may be some keyboard errors.